src/plugins/dashboard/public/application/listing/dashboard_listing.tsx

```diff
@@ -299,74 +294,57 @@ export const DashboardListing = ({
         <DashboardAppNoDataPage onDataViewCreated={() => setShowNoDataPage(false)} />
       )}
       {!showNoDataPage && (
-        <TableListView
-          createItem={!showWriteControls ? undefined : createItem}
-          deleteItems={!showWriteControls ? undefined : deleteItems}
-          initialPageSize={initialPageSize}
-          editItem={!showWriteControls ? undefined : editItem}
-          initialFilter={initialFilter ?? defaultFilter}
-          toastNotifications={core.notifications.toasts}
-          headingId="dashboardListingHeading"
-          findItems={fetchItems}
-          rowHeader="title"
-          entityNamePlural={getEntityNamePlural()}
-          tableListTitle={getTableListTitle()}
-          tableCaption={getTableCaption()}
-          entityName={getEntityName()}
-          {...{
-            emptyPrompt,
-            searchFilters,
-            listingLimit,
-            tableColumns,
+        <TableListViewKibanaProvider
+          savedObjectTaggingApi={savedObjectsTagging!}
+          applicationStart={{
```

Comment:
@clintandrewhall I've explicitly declared each of the dependency of my `applicationStart` service. This way it works but sad that TS gets in the way and that we can't just drop `core.application` for the consumer.